lighthouse-core/audits/performance-budget.js

```diff
@@ -20,6 +20,8 @@ const UIStrings = {
     =1 {1 request}
     other {# requests}
    }`,
+  /** Label for a column in a data table; entries will be how much greater the quantity of network requests were than a predetermined budget, or how much larger the size of network requests were than a predetermined budget.*/
```

Comment:
I think this is probably overkill :) What about combining the last two parts:
```suggestion
  /** Label for a column in a data table; entries will be how much the quantity or size of network requests exceeded a predetermined budget.*/
```